include/API.h

```diff
@@ -736,12 +736,23 @@ bool i2cWrite(uint8_t addr, uint8_t *data, uint16_t count);
 bool i2cWriteRegister(uint8_t addr, uint8_t reg, uint16_t value);
 
 /**
- * FILE is an integer referring to a stream for the standard I/O functions.
+ * PROS_FILE is an integer referring to a stream for the standard I/O functions.
  *
- * FILE * is the standard library method of referring to a file pointer, even though there is
+ * PROS_FILE * is the standard library method of referring to a file pointer, even though there is
  * actually nothing there.
  */
-typedef int FILE;
+typedef int PROS_FILE;
+
+
+#ifndef FILE
+/**
+ * For convenience, FILE is defined as PROS_FILE if it wasn't already defined. This provides 
+ * backwards compatability with PROS, but also allows libraries such as newlib to be incorporated
+ * into PROS projects. If you're not using C++/newlib, you can use FILE as previously

```

Comment:
I take it back, "as previously" sort of falls flat. Let's say

 > If you're not using C++/newlib, you can use FILE as [normal? as you would per the docs?]

I'm having trouble coming up with wording that I like.